mbzirc_ign/src/GameLogicPlugin.cc

```diff
@@ -79,6 +89,19 @@ class mbzirc::GameLogicPluginPrivate
     public: std::set<std::string> largeObjectsReported;
   };
 
+  /// \brief Information of target in stream
+  public: class TargetInStream
+  {
+    /// \brief Type of target: vessel, small, large
+    public: std::string type;
+
+    /// \brief Image x position
+    public: double x = 0;
```

Comment:
If these are image coordinates, should they be `unsigned int` or similar?

Reply:
yep done. [28876e6](https://github.com/osrf/mbzirc/pull/59/commits/28876e6426e82ad0c5b15c0a7e65b0d2a1e2d1d8)